forge/db/models/User.js

```diff
@@ -236,7 +251,7 @@ module.exports = {
                     return M.TeamMember.getTeamMembership(this.id, teamId, includeTeam)
                 },
                 getTeamsOwned: async function () {
-                    return M.TeamMember.getTeamsOwnedBy(this.id)
+                    return await M.TeamMember.getTeamsOwnedBy(this.id)
```

Comment:
```suggestion
                    return M.TeamMember.getTeamsOwnedBy(this.id)
```
revert temporary debugging change